test/e2e-tests-auction.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e
+
+cd dex-contracts/
+
+EXPECTED_AUCTION=1
+# Ensure no orders yet in auction slot 1
+mongo dfusion2 --eval "db.orders.find({'auctionId': ${EXPECTED_AUCTION}}).size()" | grep 0
+
+# Place 6 orders in current Auction (accountId, buyToken, sellToken, minBuy, maxSell)
+truffle exec scripts/sell_order.js 1 2 3 12 12
+truffle exec scripts/sell_order.js 2 3 2 22 20
+truffle exec scripts/sell_order.js 3 1 3 15 10
+truffle exec scripts/sell_order.js 4 1 2 180 15
+truffle exec scripts/sell_order.js 5 2 1 4 52
+truffle exec scripts/sell_order.js 6 3 1 20 280
+
+sleep 5
```

Comment:
Would love to get rid of these hard coded waits as they are really bad practice (they can be flaky and usually cause the test to take longer than necessary). 

Would appreciate any ideas on how we can know once the driver is done processing all events in the queue.

Reply:
Yes I would like that too!